pyha_analyzer/train.py

```diff
@@ -122,23 +122,22 @@ def train(model: TimmModel,
             if scheduler is not None:
                 scheduler.step()
 
-        log_pred = F.sigmoid(outputs)
-        dataset: PyhaDFDataset = data_loader.dataset # type: ignore
-        cmap, smap = map_metric(log_pred, labels, dataset.class_dist)
-        log_cmap += cmap
-        log_smap += smap
+        log_pred.append(F.sigmoid(outputs))
```

Comment:
Make sure to detach and put tensors on cpu. Otherwise this is a pusedo-memory leak on gpu since we never freed that memeory on gpu.

Reply:
oooops